site/source/user-manual/getting-started/initial-setup.rst

```diff
@@ -87,7 +87,7 @@ If you are experiencing issues with setup, try the following:
     - Mullvad - Go to "Settings -> VPN Settings -> Local Network Sharing"
     - ProtonVPN - Go to "Preferences -> Connection -> Allow LAN Connections"
 
-#. Visit or refresh (ctrl+shift+R - Linux/Windows, cmd+shift+R - Mac) the start.local page in a web browser
+#. Visit or refresh (ctrl+shift+R - Linux/Windows, cmd+option+R - Mac) the start.local page in a web browser
```

Comment:
A user has told us cmd+shift+r on a mac is "read mode" and cmd+option+R is the hard refresh

Reply:
This work for safari only. FF still uses cmd+shift+R.

Reply:
TY homer.  Updated.